src/Extension/Tasks/ESLintFixerTask.yaml

```diff
@@ -0,0 +1,25 @@
+---
+id: 'validation:frontend:eslint-fix'
+short_description: 'Fixes violations and validates your javascript code'
+help: ~
+stage: build
+version: 0.1.0
+command: 'node %sdk_dir%/frontend/libs/eslint.js --fix > %reportDir%/eslint.codestyle.json'
```

Comment:
does this tool support a config path? if yes pls add an optional option for config

Reply:
like here https://github.com/spryker-sdk/sdk/pull/82

Reply:
Fixed